Tests/SourceKitLSPTests/ClosureCompletionFormatTests.swift

```diff
@@ -0,0 +1,129 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftBasicFormat
+import SwiftParser
+import SwiftSyntax
+import Swift
+@testable import SourceKitLSP
```

Comment:
We build SourceKit-LSP with testability disabled in Swift CI because otherwise we would need to build everything twice: Once with testability enabled for testing and then with testability disabled to produce the binary we include in the toolchain. Could you change this to not use `@testable`? We use `@_spi(Testing)` to a similar effect where needed.

Reply:
Got it, updated. 🤔 I am not very familiar with how `@_spi` works, but it seems to require everything including the imports to become public. Let me know if there is another way I should have done this.